salt/states/virt.py

```diff
@@ -435,108 +628,84 @@ def running(
                     address: 192.168.0.125
 
     """
+    merged_disks = disks
+    if image:
+        default_disks = [{"system": {}}]
+        disknames = ["system"]
+        if disk_profile:
+            disklist = copy.deepcopy(
+                __salt__["config.get"]("virt:disk", {}).get(disk_profile, default_disks)
+            )
+            disknames = disklist.keys()
+        disk = {"name": disknames[0], "image": image}
+        if merged_disks:
+            first_disk = [d for d in merged_disks if d.get("name") == disknames[0]]
+            if first_disk and "image" not in first_disk[0]:
+                first_disk[0]["image"] = image
+            else:
+                merged_disks.append(disk)
+        else:
+            merged_disks = [disk]
+        salt.utils.versions.warn_until(
+            "Sodium",
+            "'image' parameter has been deprecated. Rather use the 'disks' parameter "
+            "to override or define the image. 'image' will be removed in {version}.",
+        )
 
-    ret = {
-        "name": name,
-        "changes": {},
-        "result": True,
-        "comment": "{0} is running".format(name),
-    }
+    if not update:
+        salt.utils.versions.warn_until(
+            "Magnesium",
```

Comment:
apologies to ask you this one more time. This didnt make it into neon so we need to bump this one more version. Aluminium. Once thats updated i'll be good here and we can try to get this merged. thanks

Reply:
done